### PR TITLE
Adds Meraline, anti-brute chemical

### PR DESCRIPTION
**Pull request by EgorDinamit**
## About the Pull Request

Took idea from this PR: tgstation/TerraGov-Marine-Corps/pull/384

Adds Meraline, a "tier 2" brute damage healing chemical, twice as effective as bicaridine.

## Why It's Good For The Game

Most chemicals already had "tier 2s": Kelotane has Dermaline, Dexalin has "plus" version and so on.
Only bicaridine had none, so now it does. The recipe isn't easier than that of Dermaline.

## Did you test it?

No need to.

## Changelog

:cl:
rscadd: Meraline reagent is added, a chemical effective at healing brute damage. The recipe is Bicaridine, Invaprovaline, and Iron.
/:cl:

### CODE REVIEW

**Review by Shadowtail117 — DISMISSED** [review]
Not very big on the name since most chemicals end in either -ine or -ene, and "Meralyne" sounds too much like a name than a chemical. Also, if it is meant to be a "concentrated" form of bicaridine, you might want to just label it that.

Also don't forget to add painkilling effects if you intend for it to be analgesic.

**Comment by EgorDinamit** [comment]
Updated name and description, since it was such a big concern.
And no, it isn't meant to have painkilling abilities, it is simply a "tier 2" of bicaridine, the same thing dermaline is to kelotane.

**Comment by Shadowtail117** [comment]
https://github.com/vlggms/tegustation-bay12/blob/0a0cdca3c5ae0f64acb163fb1ab5ecd956ea3850/code/modules/reagents/Chemistry-Reagents/medicines/medicines_core.dm#L33-L36

Bicaridine is analgesic which is why I brought it up.

File: code/modules/reagents/Chemistry-Reagents/medicines/medicines_core.dm [hunk 1]
```diff
@@ -69,7 +69,16 @@
 	if (alien != IS_DIONA)
 		M.heal_organ_damage(0, 12 * removed)
 
+/datum/reagent/medicine/meraline
+	name = "Meraline"
+	description = "Meraline is an advanced medicine used for brute medication. Works twice as good as bicaridine."
```
Comment (Shadowtail117) [inline code/modules/reagents/Chemistry-Reagents/medicines/medicines_core.dm]: "Brute" isn't really used in the context of injuries IC.
```suggestion
	description = "Meraline is an advanced medicine used for severe physical trauma. Works twice as good as bicaridine."
```

**Review by Shadowtail117 — DISMISSED** [review]

**Comment by EgorDinamit** [comment]
Updated description.
As I said earlier - no, meraline isn't supposed to have painkilling properties simply because it'd make bica-mera combo a bit too strong.

**Review by Shadowtail117 — APPROVED** [review]

**Pull request merged**